src/ORM/FieldType/DBMultiEnum.php

```diff
@@ -34,6 +45,15 @@ public function __construct($name = null, $enum = null, $default = null)
         }
     }
 
+    public function getValueForValidation(): array
+    {
+        $value = parent::getValueForValidation();
+        if (is_array($value)) {
+            return $value;
+        }
+        return explode(',', (string) $value);
```

Comment:
If the value isn't able to be converted to string, we'll get an error here before validation happens.
It's an edge case so probably doesn't _reeeaaallly_ matter, but if we want to be strictly correct maybe something like this:

```suggestion
        if (is_string($value) {
            return explode(',', $value);
        }
        return $value;
```

Obviously the return type then becomes `mixed` but I think it probably should be. And then if it's not iterable, the validator should complain bout that.

Reply:
I think I'd prefer a PHP exception at that point

Return type should definitely be iterable as MultiOptionFieldValidator expects an iterable value

Reply:
A PHP exception for an invalid value? That's what the validator is for? But failing that, yes please add an exception.

Reply:
Oh right sorry I misinterpreted. I've implemented your suggestion.